03-react-components/public/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Intro in React Components</title>
```

Comment:
Der Inhalt der React-Anwendung nachher wird auf Deutsch dargestellt – wollen wir das einheitlich machen? Also entweder beides auf Deutsch, oder beides auf Englisch?

Reply:
Ich lese das tatsächlich in Deutsch - ich glaube "Intro" hat sich bei mir zu sehr eingebürgert 😅 

Aber ich habe es jetzt mal in "Einführung" geändert.

Reply:
👍